cdc/entry/mounter_test.go

```diff
@@ -1615,7 +1653,18 @@ func TestBuildTableInfo(t *testing.T) {
 		handle := sqlmodel.GetWhereHandle(recoveredTI, recoveredTI)
 		require.NotNil(t, handle.UniqueNotNullIdx)
 		require.Equal(t, c.recovered, showCreateTable(t, recoveredTI))
-
+		// make sure BuildTiDBTableInfo indentify the correct primary key
+		if c.caseNumber == 5 {
```

Comment:
caseNumber can be replaced by `i` of `for i, c := range cases {`

Reply:
sure